src/connect-zbt-1/getting-started-with-thread/zbt-1-switching-from-one-thread-adapter-to-another-thread-adapter.md

```diff
@@ -0,0 +1,66 @@
+---
+zendesk:
+  article_id: 32589711910813
+  name: Migrating from another Thread adapter to Home Assistant Connect ZBT-1
+  position: 30
+  labels: connect-zbt-2, getting-started
```

Comment:
```suggestion
  labels: connect-zbt-1, getting-started
```
though I think we often use a space for ZBT-1 in the label. Not sure how consistent we are with the labels:
```suggestion
  labels: connect zbt-1, getting-started
```